lib/main.dart

```diff
@@ -16,22 +29,34 @@ class TatraSkitourApp extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    return MaterialApp(
+    return GetMaterialApp(
```

Comment:
There is also possibility to clear that using` GetMaterialApp.router()` Then you can create routing somewhere else and pass the whole class inside it.